draft-ietf-httpbis-client-hints.md

```diff
@@ -50,6 +50,7 @@ normative:
       organization: Google
   W3C.REC-html5-20141028:
   W3C.CR-css-values-3-20150611:
+#  W3C.REC-CSS2-20110607:
```

Comment:
@reschke @mnot I get a crash when I try to build with this. A bug in the fetched ref?

```
Error at xsl:value-of on line 2278 of rfc2629.xslt:
  SERE0014: Illegal HTML character: decimal 135
  at xsl:apply-templates (file:/Users/igrigorik/git/specs/http-extensions/lib/rfc2629.xslt#2519)
     processing /rfc/back[1]/references[1]/reference[10]
  at xsl:apply-templates (file:/Users/igrigorik/git/specs/http-extensions/lib/rfc2629.xslt#1958)
     processing /rfc/back[1]/references[1]
  at xsl:apply-templates (file:/Users/igrigorik/git/specs/http-extensions/lib/rfcbootstrap.xslt#150)
     processing /rfc/middle[1]
  at xsl:call-template name="body" (file:/Users/igrigorik/git/specs/http-extensions/lib/rfc2629.xslt#2667)
  at xsl:apply-templates (file:/Users/igrigorik/git/specs/http-extensions/lib/rfc2629.xslt#3700)
     processing /rfc
Illegal HTML character: decimal 135
Makefile:81: recipe for target 'draft-ietf-httpbis-client-hints.html' failed
```